pkg/controller/endpoint/endpoints_controller.go

```diff
@@ -70,6 +70,10 @@ const (
 	TolerateUnreadyEndpointsAnnotation = "service.alpha.kubernetes.io/tolerate-unready-endpoints"
 )
 
+var (
```

Comment:
Please avoid global variables. 
This should be a field in endpointsController.

Reply:
Done 👍